apps/drupal/templates/misc/status-messages.html.twig

```diff
@@ -0,0 +1,56 @@
+{#
+/**
+ * @file
+ * Theme override for status messages.
+ *
+ * Displays status, error, and warning messages, grouped by type.
+ *
+ * An invisible heading identifies the messages for assistive technology.
+ * Sighted users see a colored box. See http://www.w3.org/TR/WCAG-TECHS/H69.html
+ * for info.
+ *
+ * Add an ARIA label to the contentinfo area so that assistive technology
+ * user agents will better describe this landmark.
+ *
+ * Available variables:
+ * - message_list: List of messages to be displayed, grouped by type.
+ * - status_headings: List of all status types.
+ * - display: (optional) May have a value of 'status' or 'error' when only
+ *   displaying messages of that specific type.
+ * - attributes: HTML attributes for the element, including:
+ *   - class: HTML classes.
+ */
+#}
+
+{% set status_heading = {
+  'status': 'Status message'|t,
+  'error': 'Error message'|t,
+  'warning': 'Warning message'|t,
+  'info': 'Informative message'|t,
+  }
+%}
+
+{% for type, messages in message_list %}
```

Comment:
Are we printing all out on the page as part of the demo in Drupal?

Reply:
Yeah, but none really exist. I'd been adding them via preprocess to test to make sure it could handle them individually or in bulk. But really, this should handle any error we give it in any order.

Reply:
@jryanconklin : let's do this: in the .theme file, fire off a warning, error, and info message. Maybe have it fire on every single page load just to show it off, informing users to comment remove it.